Adyen/Core/APIClient/Requests/OrderStatusRequest.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  OrderStatusRequest.swift
+//  Adyen
+//
+//  Created by Mohamed Eldoheiri on 5/6/21.
+//  Copyright © 2021 Adyen. All rights reserved.
+//
+
+import Foundation
+
+/// :nodoc:
+public struct OrderStatusRequest: Request {
```

Comment:
We are not exposing this to Jazzy? 
At least we can add proper comments from [API DOCS](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/orders)

Reply:
it shouldn't be exposed to jazzy, yes